### DISCUSSION TITLE
Cannot properly convert the notebook into pdf

### DISCUSSION THREAD

**Discussion by sorobedio** [seominjoon/kaist-ai605#84] [Q&A] [answered]
The only option i have to convert the notebook into pdf file is print but that option cut out some part of the notebook.
how to get a proper pdf file? (i am using colab)

**Comment by Yassawe** [comment, marked as answer]
Have you tried this?

https://stackoverflow.com/questions/52588552/google-co-laboratory-notebook-pdf-download

**Reply by sorobedio** [reply]
thank you. i did not know that. i was so lazy i did not even search for on google.
